pyro_api/__init__.py

```diff
@@ -0,0 +1,11 @@
+from pyro_api.dispatch import distributions, handlers, infer, ops, optim, pyro, pyro_backend
```

Comment:
pep8 [discourages](https://www.python.org/dev/peps/pep-0008/#package-and-module-names) use of underscores in package names. Do you think it's worth renaming this to something like `pyroapi`, or should we disregard pep8?

Reply:
It was looking a bit odd for sure. `pyroapi` seems good to me, happy to change to that instead.

Reply:
Yeah, let's make the package named `pyroapi`.